stdlib/opal-parser.rb

```diff
@@ -12,9 +12,11 @@ def eval(str)
     compiling_options = __OPAL_COMPILER_CONFIG__.merge(default_eval_options)
     code = Opal.compile str, compiling_options
     %x{
-      return (function(self) {
-        return eval(#{code});
-      })(self)
+      return (
+        Function(
```

Comment:
You can add parameters to Function by declaring them in the first argument like so: "self,other,anything".